src/shared/components/oncoprint/ResultsViewOncoprint.tsx

```diff
@@ -221,6 +225,32 @@ export default class ResultsViewOncoprint extends React.Component<IResultsViewOn
                 getBrowserWindow().globalStores.routing.updateRoute(newParams, undefined, true, true);
             }
         );
+        
+        if (this.props.replaceRoutingHistoryCallback) {
+            this.silentUrlParamsReaction = reaction(
```

Comment:
could this go in the reaction up above (line 200)

Reply:
Well, I have added this new reaction because the final functionality is not to trigger refresh of components when the url is updated for certain url parameters (such as _oncoprint_sortby_).  If your changes in the future accomodate this, I agree.